client/public/employeePage/employee.js

```diff
@@ -0,0 +1,90 @@
+const addHomeAddressForm = document.getElementById("addHomeAddressForm");
+const vacationRequestForm = document.getElementById("vacationRequestForm")
+let tbody=document.getElementById("tbody");
+let messages=document.getElementById("messages");
+
+Date.prototype.toDateInputValue = (function() {
+    let local = new Date(this);
+    local.setMinutes(this.getMinutes() - this.getTimezoneOffset());
+    return local.toJSON().slice(0,10);
+});
+
+document.getElementById('vacationRequestFrom').value = new Date().toDateInputValue();
+document.getElementById('vacationRequestTo').value = new Date().toDateInputValue();
+
+//fetch user function
+const display = async() => {
+    try
+    {
+        const response = await fetch("http://localhost:4000/api/employeeInfo?id=1", {method: "GET"})
```

Comment:
URL should be dynamic as it depends on the environment
please use a dynamic string. example:
``http://${host}/api/...``